lib/resources/example_json/r4_examples_account.rb

```diff
@@ -0,0 +1,39 @@
+module Cerner
+  module Resources
+
```

Comment:
You need a Search Bundle JSON for your search example

Reply:
It was updated to the wrong resource: https://github.com/cerner/fhir.cerner.com/commit/62764511c7f719faa5141a6b7bcbfcc839226e9f#diff-139e4f82fa560c1b721301666faf1a3eR57

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/e2c16b32dcfacbc09c4d7405900bfeb259552714#diff-139e4f82fa560c1b721301666faf1a3e

Reply:
+1